src/Exception/BadRequestTokenHttpException.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+/**
+ * This file is part of Contao.
+ *
+ * Copyright (c) 2005-2015 Leo Feyer
+ *
+ * @license LGPL-3.0+
+ */
+
+namespace Contao\CoreBundle\Exception;
+
+/**
+ * This exception is thrown when there is an issue with the request token.
+ *
+ * FIXME: obsolete when using symfony security?
+ *
+ * @author Christian Schiffler <https://github.com/discordier>
+ */
+class BadRequestTokenHttpException extends BadRequestHttpException
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function __construct(
+        $message = 'Invalid request token. Please <a href="javascript:window.location.href=window.location.href">go back</a> and try again.',
```

Comment:
maybe your code does a force-reload. And it would probably also reload the top frame instead of an iframe (which is better in theory). Guess we should live with that, though I don't like the button name. It should say `reload the page` ;-)